### ISSUE TITLE
Incorrect translate3d if not at top of page.

### ISSUE THREAD

**Issue by trinadin** [tomsarduy/parallux#3]
I was trying to incorporate this on an element that falls just below a fixed header. I ended up adding an element that is the same height as the header to hold space, and found out that the translate3d that is initially set on parallux-inner is incorrect if the section that you wish to parallax is not at the top of the page. You can recreate this in demo3 by adding a div of any height above the top parallax item. It will incorrectly calculate, resulting in the video being pulled up and out of the frame. Any suggestions?

**Comment by tomsarduy** [comment]
Yes, I'm aware of that. Now I'm remembering that case was really difficult to achieve because `.parallux-inner` need to have extra space in top and bottom for the parallax animation with the background image, so I ended only supporting full page parallax and fixed header. 

In your specific case, I could make the `translate3d` start in the right position, but when the background image animate, you will see a white space above the parallax element when you start scrolling.

PD: I didn't know people were still using my old plugin. I will work on some improvements soon.

**Comment by trinadin** [comment]
Thanks for the response. I ended up doing a work around that kept it at the top of the page, but raised the height to include the fixed header height, and added some blank space within parallux-inner to push the content within down a bit. This resulted in it calculating at 0, 0, 0, which seems to be working out well.

Your plugin is definitely the best that I've tried, and it comes up very high in search results for common parallax searches. I'm utilizing it on my latest project, and so far so good. It'll likely go into production. Thank you for making it, and I very much look forward to an updated version!
